app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
```

Comment:
```suggestion
    def __init__(
        self,
        comfort_class: int,
        clean_mark: int,
        brand: str
) -> None:
```
Fix indentations like in the example to increase the readability of your code. Fix everywhere